### DISCUSSION TITLE
Audio is not playing

### DISCUSSION THREAD

**Discussion by Alianjum** [jspsych/jsPsych#1154] [Q&A]
I am having an issue with this plugin " jspsych-categorize-audio ".
The audio is not plyaing with the options 
can you guid me whats the issue in this?

you can check it here
https://www.sertechs.com/Heaton/
I have attached the file of the in the below link

https://drive.google.com/file/d/19iM5d_YCwFcc7IZ8fDdFpKZAxxCdvhLh/view?usp=sharing
 
Please help me in this issue 
i really thankfull to you

**Comment by jodeleeuw** [comment]
Hi @Alianjum ,

It looks like the custom plugin you are using, `jspsych-categorize-audio`, doesn't actually play the audio file. 

In the `audio-keyboard-response` this chunk of code is necessary to play the audio file. 

https://github.com/jspsych/jsPsych/blob/d6217106256be5e5eeb4fe29c93cdd3fa9021878/plugins/jspsych-audio-keyboard-response.js#L146-L152

I think you'll need to add this to your custom plugin to get it to play the audio.
